recipes/full_finetune_distributed.py

```diff
@@ -547,6 +548,10 @@ def _setup_model(
 
         # Apply tensor parallelism to the model
         if self.parallel_dims.tp_enabled:
+            if self._enable_fp8_training:
+                raise ValueError(
+                    "Float8 training + tensor parallelism is not supported yet"
+                )
```

Comment:
Regarding this, are there still known gaps that need to be addressed? E.g. of the three issues from @vkuzo's [comment](https://github.com/pytorch/torchtune/pull/2404#issuecomment-2685709722) it seems like one is closed, one was a red herring, and one is not relevant to us today (as it's related to async TP). Wondering if we're good to just gate on nightly PyTorch instead of disabling entirely?

Reply:
the main issue I was worried about earlier turned out to be due to bad hardware, so we are all good!
 
are there plans / needs to support asyncTP in torchtune?

Reply:
@vkuzo asyncTP is not something that has really been on our radar. But given that it shows some nice speedups, I think it would be worth exploring.